prow/config/config_test.go

```diff
@@ -1370,13 +1361,9 @@ postsubmits:
 			expectError: true,
 		},
 		{
-			name: "overwrite PodNamespace",
-			prowConfig: `
-pod_namespace: test`,
-			jobConfigs: []string{
-				`
-pod_namespace: debug`,
-			},
+			name:               "overwrite PodNamespace",
+			prowConfig:         `pod_namespace: test`,
+			jobConfigs:         []string{},
```

Comment:
oh this is for testing we don't overwrite prow config from job config - we can get rid of this testcase now as strictparse won't allow we set prow config fields in job config.

Reply:
ok removing it :+1:

Reply:
done